web-console/future-openshift/search/search.md

```diff
@@ -0,0 +1,58 @@
+# Search page
+The Search page will be updated to have a variety of new features.
+
+## View multiple resources
+![1](img/empty-state.png)
+* The resource selection dropdown will be a filter. When no resources are selected, an empty state will tell users to select resources to get started. It will also describe some of the functionality of the page.
+
+![1](img/expanded-list0.png)
+* Users can view more than one type of resource by clicking the dropdown and selecting resource types from the list.
+* There will be a max number of resources that users can select at a time, which is yet to be determined.
+
+![2](img/expanded-list1.png)
```

Comment:
Very cool addition to be able to create a resource type from the Search results!